addons/mrp/views/product_views.xml

```diff
@@ -83,6 +86,39 @@
             <field name="inherit_id" ref="stock.product_template_form_view_procurement_button"/>
             <field name="groups_id" eval="[(4, ref('mrp.group_mrp_user'))]"/>
             <field name="arch" type="xml">
+                <xpath expr="//button[@name='action_open_quants']" position="replace">
+                    <button class="oe_stat_button"
+                            name="action_open_quants"
+                            icon="fa-cubes"
+                            type="object" attrs="{'invisible':['|', ('type', '=', 'service'), ('type', '=', 'consu'), ('is_kits', '=', False)]}">
+                            <div class="o_field_widget o_stat_info">
+                                <span class="o_stat_value">
+                                    <field name="qty_available" widget="statinfo" nolabel="1" class="mr4"/>
+                                    <field name="uom_name"/>
+                                </span>
+                                <span class="o_stat_text" attrs="{'invisible': [('is_kits', '=', True)]}">On Hand</span>
+                                <span class="o_stat_text" attrs="{'invisible': [('is_kits', '=', False)]}">Available</span>
+                            </div>
+                    </button>
+                </xpath>
+                <xpath expr="//button[@name='action_product_tmpl_forecast_report']" position="attributes">
+                    <attribute name="attrs">{'invisible':['|', ('type', '!=', 'product'), ('is_kits', '=', True)]}</attribute>
+                </xpath>
+                <xpath expr="//button[@name='action_product_tmpl_forecast_report']" position="after">
+                    <button type="object"
+                        name="action_open_quants"
+                        attrs="{'invisible':[('is_kits', '=', False)]}"
+                        context="{'default_product_tmpl_id': id}"
+                        class="oe_stat_button" icon="fa-cubes">
+                        <div class="o_field_widget o_stat_info">
+                            <span class="o_stat_value">
+                                <field name="virtual_available" widget="statinfo" nolabel="1" class="mr4"/>
+                                <field name="uom_name"/>
+                            </span>
+                            <span class="o_stat_text">Forecasted Available</span>
+                        </div>
+                    </button>
+                </xpath>
```

Comment:
Do a compute that display or not the stat button for product quantity instead of replacing all the xml. It would avoid bridge module like sale_rental_mrp. And it would be easier and safer to manage it this way